apps/L1G/restaurant/restaurant-frontend/src/components/auth/update-user-profile/UpdateUserProfile.tsx

```
@@ -0,0 +1,103 @@
+'use client';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { useUpdateUserMutation } from '@/generated';
+import { passwordSchema, phoneSchema } from '@/utils/UpdateUserUtils';
```

ESLint

[error] 'passwordSchema' is defined but never used.

(no-unused-vars)